index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Mobile responsive design</title>
+    <meta charset='utf-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <link rel="stylesheet" href="Styles/main.css">
+  </head>
+  <body>
+    <div class='c-container'>
+      <div class='c-container__banner'>
+          <h1>Play the Rock Paper Scissors Game!</h1>
+      </div>
+      <div class='c-container__main'>
+        <div class="c-main__box-1">
```

Comment:
You could wrap these 3 inside a div and just hide that one div, rather than hiding each image individually